docs/projects/intro.md

```diff
@@ -35,6 +35,21 @@ The following table details the relevance of each item in the project tree:
 
 The remaining structure of the project, and each subproject, depends on the type of service under development.
 
+## TOSCA Additions
+
+In addition to the normal project structure, you may also include a `TOSCA-Metadata` directory to be included in the final package. This directory is expected to have a single `TOSCA.meta` file in.
```

Comment:
why do we have to have a directory with just a single file in it? will this be added to in future?

Reply:
No idea. It's part of the Tosca spec, so we'd only add to it if they add anything to it.